packages/fast-components-react-msft/src/pivot/pivot.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Pivot",
+    "description": "A pivot component's schema definition.",
+    "type": "object",
+    "id": "pivot",
+    "properties": {
+        "activeId": {
+            "title": "Active pivot ID",
+            "type": "string"
+        },
+        "label": {
+            "title": "HTML aria-label attribute",
+            "type": "string"
+        },
+        "orientation": {
+            "title": "HTML aria-orientation attribute",
+            "type": "string",
+            "enum": [
+                "horizontal",
+                "vertical"
+            ]
+        },
+        "items": {
+            "title": "Items",
+            "type": "array",
+            "items": {
+                "title": "Item",
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "title": "HTML id attribute",
+                        "type": "string"
+                    }
+                },
+                "required": [
+                    "id"
+                ],
+                "reactProperties": {
+                    "tab": {
+                        "title": "Pivot tab",
+                        "type": "children",
+                        "pluginId": "@microsoft/fast-components-react-msft/pivot/items/tab"
+                    },
+                    "content": {
+                        "title": "Pivot content",
+                        "type": "children",
+                        "pluginId": "@microsoft/fast-components-react-msft/pivot/items/content"
+                    }
+                }
+            }
+        }
+    },
+    "required": [
+        "tab",
+        "content"
+    ]

```

Comment:
This is in the wrong location, it should be on the same level as id in items, so you could add it to that array. If it errors when you do that (because the data utilities may not know what to do to generate a required child) then I would remove it altogether.